flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/TableSinkTest.scala

```diff
@@ -623,6 +623,32 @@ class TableSinkTest extends TableTestBase {
     // source and sink has same parallelism, but sink shuffle by pk is enforced
     util.verifyExplain(stmtSet, ExplainDetail.JSON_EXECUTION_PLAN)
   }
+
+  @Test
+  def testManagedTableSinkWithDisableCheckpointing(): Unit = {
```

Comment:
Can you add more tests:
- streaming mode with cp
- streaming mode without cp
- batch mode